### PR TITLE
Update docs.md

### PR DESCRIPTION
**Pull request by mileyvirus**
I added the fact that twig debugging is disabled by default.

### CODE REVIEW

File: pages/08.advanced/03.debugging/docs.md [hunk 1]
```diff
@@ -53,7 +53,13 @@ $grav['debugger']->addMessage($this)
 
 ### Dump command for Twig
 
-You can also display Twig variables from your Twig templates.  This is done in a similar fashion, but the results are displayed in the **Messages** panel of the Debug Bar.
+You can also display Twig variables from your Twig templates.  This is done in a similar fashion, but the results are displayed in the **Messages** panel of the Debug Bar. This feature is **disabled** by default, but can be turned on either globally or for your [development environment](../environment-config) only via the `system.yaml` configuration file:
+
+```
+twig:
+  cache: true
```
Comment (rhukster) [inline pages/08.advanced/03.debugging/docs.md]: can you remove this `cache: true` line? it's not needed here.

**Comment by mileyvirus** [comment]
Sure! Thanks for teaching

File: pages/08.advanced/03.debugging/docs.md (hunk 1 above)
Comment (mileyvirus) [inline pages/08.advanced/03.debugging/docs.md]: Sure! Thanks for teaching.

**Comment by rhukster** [comment]
Thanks for the contribution!

**Pull request merged**